### PR TITLE
No sentient software, and other spelling

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: qml/firstrun/Tutorial.qml [hunk 1]
```diff
@@ -56,9 +56,9 @@ Item {
 
     Component.onCompleted: {
         centerIcon.name = "logo-asteroidos"
-        //% "Welcome !"
+        //% "Welcome."
         title.text = qsTrId("id-tutorial-welcome-title") + localeManager.changesObserver
-        //% "Let me guide you."
+        //% "This is AteroidOS."
```
Comment (FlorentRevest) [inline qml/firstrun/Tutorial.qml]: AteroidOS?

**Comment by FlorentRevest** [comment]
Modifying source strings means you'd need to run: lupdate-qt5 asteroid-launcher.pro
And also adapt i18n/asteroid-launcher.en_GB.ts

+ Can we have more details as to why you believe this would be better?

File: qml/firstrun/Tutorial.qml [hunk 2]
```diff
@@ -56,9 +56,9 @@ Item {
 
     Component.onCompleted: {
         centerIcon.name = "logo-asteroidos"
-        //% "Welcome !"
+        //% "Welcome."
         title.text = qsTrId("id-tutorial-welcome-title") + localeManager.changesObserver
-        //% "Let me guide you."
+        //% "This is AsteroidOS."
```
Comment (comradekingu) [inline qml/firstrun/Tutorial.qml]: This is not creepy

File: qml/firstrun/Tutorial.qml [hunk 3]
```diff
@@ -71,7 +71,7 @@ Item {
                     centerIcon.name = "ios-move"
                     //% "Indicators"
                     title.text = qsTrId("id-tutorial-indicators-title") + localeManager.changesObserver
-                    //% "Show you possible gestures."
+                    //% "show you possible gestures."
```
Comment (comradekingu) [inline qml/firstrun/Tutorial.qml]: lowercase since it is in a sentence started earlier.

File: qml/firstrun/Tutorial.qml [hunk 4]
```diff
@@ -208,9 +208,9 @@ Item {
                 ScriptAction {
                     script: {
                         centerIcon.name = "ios-arrow-round-down"
-                        //% "Applications"
+                        //% "Apps"
```
Comment (comradekingu) [inline qml/firstrun/Tutorial.qml]: This is just shorter, without any loss.

File: qml/firstrun/Tutorial.qml [hunk 5]
```diff
@@ -316,7 +316,7 @@ Item {
                         centerIcon.name = "ios-log-out"
                         //% "Leave an app"
                         title.text = qsTrId("id-tutorial-leaveapp-title") + localeManager.changesObserver
-                        //% "With a left to right gesture"
+                        //% "following a rightward gesture"
```
Comment (comradekingu) [inline qml/firstrun/Tutorial.qml]: There is no other way to go rightwards, except from the left.

**Comment by FlorentRevest** [comment]
Merged manually with https://github.com/AsteroidOS/asteroid-launcher/commit/40f3c32cc7a99d8b99c9d06b979d2bdaa9856fb9

**Pull request closed without merging**